cleanlab/datalab/datalab.py

```diff
@@ -90,6 +90,9 @@ class Datalab:
     label_name : str, optional
         The name of the label column in the dataset.
 
+        In the case of classification (both multiclass and multilabel), the labels are internally mapped to integers in a [0, 1, ..., K-1] format,
+        ordered lexicographically by class name. Make sure that your model's predictions reflect this ordering.
```

Comment:
this should be for pred_probs. Why does user need to know about this for their labels?

Reply:
You're right, it's sufficient to just point this out for the pred_probs.

Reply:
Removed.